zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/metrics/PersistedUsageMetrics.java

```diff
@@ -122,4 +118,13 @@ public PersistedUsageMetrics recordTU(final String tenantId, final Long assignee
     }
     return this;
   }
+
+  public PersistedUsageMetrics close(final long time) {
+    if (getFromTime() == TIME_NOT_SET) {
+      // initialize fromTime in case metrics were recorded before the first checker run
```

Comment:
🔧 this hasn't to do something with the checker, right? This can happen if
- there has been no metric recorded so far that has triggered bucket creation
- there has been no export run so far that has reset and created a new bucket
right?

Reply:
Same, I meant the full chain checker->processor->applier.

Reply:
understood. let's be super clear as there is a `checker` class and it's easy to mistake that.